specutils/io/default_loaders/sdss_v.py

```diff
@@ -347,9 +351,10 @@ def load_sdss_spec_1D(file_obj, *args, hdu: Optional[int] = None, **kwargs):
 
 
 @data_loader(
-    "SDSS-V spec multi",
+    "SDSS-V spec",
     identifier=spec_sdss5_identify,
     dtype=SpectrumList,
+    force=True,
```

Comment:
With the `multi` removed, what is the difference between this data loader and the above data loader at line 318?   Same with the `mwm` data loaders?  

With this removal, what would be the correct `format` to specify to load all data extensions?  Is it now the default?  Jdaviz only supports using `format` to during data load to provide hints to the type of filepath.

Reply:
My goal with this was to simplify the interface. The difference previously was that `multi` loaded every valid HDU extension, whereas the single `SpectrumList` ones would load just a specified HDU. Since it's a `SpectrumList`, I felt the default loader for that type should just load every extension.

Across both the `Spectrum1D` and `SpectrumList` loaders there should be a single format (`SDSS-V [DATATYPE]`), which should be detected automagically and no longer needs to be manually specified on `SpectrumList`.

Reply:
I'm planning to catch up on this this afternoon, I might have thoughts then.